docs/section22/Rule22-28.md

```diff
@@ -0,0 +1,52 @@
+
+# CHW&CW - Rule 22-28  
+
+**Schema Version:** 0.0.10 **Mandatory Rule:** True
+**Rule ID:** 22-28  
+**Rule Description:** Each baseline chiller shall be modeled with separate condenser-water pump interlocked to operate with the associated chiller.  
+**Rule Assertion:** B-RMR = expected value  
+**Appendix G Section:** Section 22 CHW&CW Loop  
+**90.1 Section Reference:** Section G3.1.3.11 Heat Rejection (Systems 7, 8, 11, 12, and 13)  
+**Data Lookup:** None  
+**Evaluation Context:** Building  
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system that is Type-7, 8, 11.1, 11.2, 12, 13, 7b, 8b, 11b, 12b, 13b.
+
+**Function Calls:**  
+
+1. get_baseline_system_types()
+
+**Applicability Checks:**  
+
+- Get B-RMR system types: `baseline_hvac_system_dict = get_baseline_system_types(B-RMR)`
+
+  - Check if B-RMR is modeled with at least one air-side system that is Type-7, 8, 11.1, 11.2, 12, 13, 7b, 8b, 11b, 12b, 13b, i.e. with air-side system served by chiller(s), continue to rule logic: `if any(sys_type in baseline_hvac_system_dict.keys() for sys_type in ["SYS-7", "SYS-8", "SYS-11.1", "SYS-11.2", "SYS-12", "SYS-13", "SYS-7B", "SYS-8B", "SYS-11B", "SYS-12B", "SYS-13B"]): CHECK_RULE_LOGIC`
+
+  - Else, rule is not applicable to B-RMR: `else: RULE_NOT_APPLICABLE`
+
+## Rule Logic:
+
+- For each pump in B_RMR: `for pump_b in B_RMR.RulesetModelInstance.pumps:`
+
+  - Save loop served by pump and pump to a dictionary: `loop_pump_dictionary[pump_b.loop_or_piping].append(pump)`
+
+- For each chiller in B_RMR: `for chiller_b in B_RMR.RulesetModelInstance.chillers:`
+
+  - Check if condensing loop is connected to other chiller, raise fail flag: `if chiller_b.condensing_loop in condensing_loop_array: fail_flag = TRUE`
```

Comment:
I am confused with this check. Does it suggest that there should be as many CW loops as chillers? I am not sure how this would be modeled.

Reply:
Maybe this is verifying that there is only one condenser loop serving a chiller, else it results in fail.  Should discuss.

Reply:
I am not sure. Seems like it's verifying that there is no more than one chiller on each condenser water loop, but don't we want one condenser water loop per project, just like we want one primary and once secondary CHW loop per project?

Reply:
Updated.